modules/oadp-configuring-aws-bsl.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-aws-bsl.adoc_{context}"]
+= Configuring the backup storage location using AWS
+
+You can configure the {aws-short} backup storage location (BSL) as shown in the following example.
+
+.Prerequisites
+
+* You have created an object storage bucket using {aws-short}.
+* You have installed the {oadp-short} Operator.
+
+.Procedure
+
+. Configure the BSL custom resource (CR) as shown in the following example:
```

Comment:
Since this is the only step in the procedure, this should be an unnumbered list item [per the SSG](https://redhat-documentation.github.io/supplementary-style-guide/#single-step-procedures):
```suggestion
* Configure the BSL custom resource (CR) as shown in the following example:
```